### PR TITLE
feat(connector) : add template code for connector nexinets

### PR DESCRIPTION
**Pull request by srujanchikke**
## Type of Change

- [ ] Bugfix
- [x] New feature
- [ ] Enhancement
- [ ] Refactoring
- [ ] Dependency updates

## Description

Template code for Nexinets

### Additional Changes

- [ ] This PR modifies the database schema
- [x] This PR modifies application configuration/environment variables

## Motivation and Context

Raising PR for template file alone for new connector, so that flows PR can be easily reviewed

## How did you test it?

testing not required for template code

## Checklist

- [x] I formatted the code `cargo +nightly fmt --all`
- [x] I addressed lints thrown by `cargo clippy`
- [x] I reviewed submitted code
- [ ] I added unit tests for my changes where possible
- [ ] I added a [CHANGELOG](/CHANGELOG.md) entry if applicable

### CODE REVIEW

File: crates/router/src/connector/nexinets/transformers.rs [hunk 1]
```diff
@@ -0,0 +1,208 @@
+use masking::Secret;
+use serde::{Deserialize, Serialize};
+
+use crate::{
+    connector::utils::PaymentsAuthorizeRequestData,
+    core::errors,
+    types::{self, api, storage::enums},
+};
+
+//TODO: Fill the struct with respective fields
```
Comment (jagan-jaya) [inline crates/router/src/connector/nexinets/transformers.rs:10]: remove todo

File: crates/router/src/connector/nexinets/transformers.rs [hunk 2]
```diff
@@ -0,0 +1,208 @@
...
+                    amount: item.request.amount,
+                    card,
+                })
+            }
+            _ => Err(errors::ConnectorError::NotImplemented("Payment methods".to_string()).into()),
+        }
+    }
+}
+
+//TODO: Fill the struct with respective fields
```
Comment (jagan-jaya) [inline crates/router/src/connector/nexinets/transformers.rs:50]: remove todo

File: crates/router/src/connector/nexinets/transformers.rs [hunk 3]
```diff
@@ -0,0 +1,208 @@
...
+    fn from(item: NexinetsPaymentStatus) -> Self {
+        match item {
+            NexinetsPaymentStatus::Succeeded => Self::Charged,
+            NexinetsPaymentStatus::Failed => Self::Failure,
+            NexinetsPaymentStatus::Processing => Self::Authorizing,
+        }
+    }
+}
+
+//TODO: Fill the struct with respective fields
```
Comment (jagan-jaya) [inline crates/router/src/connector/nexinets/transformers.rs:88]: remove todo

File: crates/router/src/connector/nexinets/transformers.rs [hunk 4]
```diff
@@ -0,0 +1,208 @@
...
+                redirection_data: None,
+                mandate_reference: None,
+                connector_metadata: None,
+            }),
+            ..item.data
+        })
+    }
+}
+
+//TODO: Fill the struct with respective fields
```
Comment (jagan-jaya) [inline crates/router/src/connector/nexinets/transformers.rs:121]: remove todo

File: crates/router/src/connector/nexinets/transformers.rs [hunk 5]
```diff
@@ -0,0 +1,208 @@
...
+        match item {
+            RefundStatus::Succeeded => Self::Success,
+            RefundStatus::Failed => Self::Failure,
+            RefundStatus::Processing => Self::Pending,
+            //TODO: Review mapping
+        }
+    }
+}
+
+//TODO: Fill the struct with respective fields
```
Comment (jagan-jaya) [inline crates/router/src/connector/nexinets/transformers.rs:160]: remove todo

File: crates/router/src/connector/nexinets/transformers.rs [hunk 6]
```diff
@@ -0,0 +1,208 @@
...
+            response: Ok(types::RefundsResponseData {
+                connector_refund_id: item.response.id.to_string(),
+                refund_status: enums::RefundStatus::from(item.response.status),
+            }),
+            ..item.data
+        })
+    }
+}
+
+//TODO: Fill the struct with respective fields
```
Comment (jagan-jaya) [inline crates/router/src/connector/nexinets/transformers.rs:201]: remove todo

**Review by jagan-jaya — CHANGES_REQUESTED** [review]

**Review by ArjunKarthik — APPROVED** [review]

**Review by jagan-jaya — APPROVED** [review]

**Review by jarnura — APPROVED** [review]

**Review by ArjunKarthik — APPROVED** [review]

**Review by jagan-jaya — APPROVED** [review]

**Review by ArjunKarthik — APPROVED** [review]

**Review by jarnura — APPROVED** [review]

**Pull request merged**